tests/test_encoding/test_count_frequency_encoder.py

```diff
@@ -122,10 +122,28 @@ def test_error_if_input_df_contains_categories_not_present_in_fit_df(
 ):
     # test case 3: when dataset to be transformed contains categories not present in
     # training dataset
-    with pytest.warns(UserWarning):
-        encoder = CountFrequencyEncoder()
-        encoder.fit(df_enc)
-        encoder.transform(df_enc_rare)
+    msg = "During the encoding, NaN values were introduced in the feature(s) var_A."
```

Comment:
this test is only testing the default parameter `rare_labels='ignore`'. We never enter rare_labels='raise'. We need to modify the test slightly.